modules/renter/downloadqueue.go

```diff
@@ -60,6 +60,9 @@ func (r *Renter) DownloadQueue() []modules.DownloadInfo {
 			StartTime:   d.startTime,
 		}
 		downloads[i].Received = atomic.LoadUint64(&d.atomicDataReceived)
+		if d.downloadErr != nil {
+			downloads[i].Error = d.downloadErr.Error()
+		}
```

Comment:
That is true except for one case: when the file doesn't exist.
There is a TODO in `Download` that reads:
```
// TODO: Eventually just return the channel to the error instead of the
// error itself.
select {
	case err := <-d.downloadFinished:
	return err
```

Perhaps we should tackle this now. There aren't many callsites to update. Then the "file not found" error would look like:
```go
	lockID := r.mu.RLock()
	file, exists := r.files[path]
	r.mu.RUnlock(lockID)
	if !exists {
		ch := make(chan error, 1)
		ch <- errors.New("no file with that path")
		return ch
	}
```